contracts/contracts/chains/polygon_zkevm/PolygonZkEVMMessengers.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import "../../../interfaces/IMessenger.sol";
+import "../../RestrictedCalls.sol";
+import "./PolygonZkBridgeInterface.sol";
+
+// This messenger is used by both L1 & L2 messengers.
+// local means it is on the same chain and remote means
+// it is on opposite chain.
```

Comment:
`The prefix local refers to instances on the same chain, while remote refers to instances on the other layer.`